src/Rector/Controller/RenderZendView.php

```diff
@@ -115,22 +115,20 @@ public function refactor(Node $node): ?Node
 
     private function classMethodIsEmpty(ClassMethod $classMethod) {
         if (count($classMethod->stmts) == 0) {
-            $classNode = $this->$currentClassMethod->getAttribute(AttributeKey::CLASS_NODE);
+            $classNode = $classMethod->getAttribute(AttributeKey::CLASS_NODE);
             $this->symfonyStyle->caution('empty classMethod "'.$classMethod->name.'" in "'. $classNode->name .'" on line '. $classMethod->getLine());
             return true;
         }
         return false;
     }
 
-    private $currentClassMethod;
     private function detectReturnStatements(ClassMethod $classMethod)
     {
-        $this->$currentClassMethod = $classMethod;
```

Comment:
obacht: hier hattest du vorher eine variable variable.. - man beachte das "zusätzliche `$`" innerhalb von `$this->$currentClassMethod` - das war sicher keine absicht, aber ist seeeehr gefährlich

siehe https://www.php.net/manual/en/language.variables.variable.php